File: Python/Niveau 1/7 – Conditions avancées, opérateurs booléens/Personne disparue.py
numP = int(input())
Lsize = int(input())
kata = False

for i in range(Lsize):
   num = int(input())   
   if num == numP:
      kata = True
      
if kata:
   print("Sorti de la ville")
else:
   print("Encore dans la ville")

File: Pascal/Niveau 1/7 – Conditions avancées, opérateurs booléens/Personne disparue.pas
program Solution;
var
   i, IDP, listSize, num: LongInt;
   hasLeft: boolean;
begin
   hasLeft := false;
   read(IDP, listSize);
   for i := 1 to listSize do
   begin
      read(num);
      if num = IDP then
      begin
         hasLeft := true;
      end;
   end;
   if hasLeft = true then
   begin
      writeln('Sorti de la ville');
   end
   else
   begin
      writeln('Encore dans la ville');
   end;
end.